pybammeis/eis_simulation.py

```diff
@@ -94,10 +100,10 @@ def set_up_model_for_eis(self, model):
         # Create a voltage variable
         V_cell = pybamm.Variable("Voltage variable [V]")
         new_model.variables["Voltage variable [V]"] = V_cell
-        V = new_model.variables["Voltage [V]"]
+        V = new_model.variables[target]
         # Add an algebraic equation for the voltage variable
         new_model.algebraic[V_cell] = V_cell - V
-        new_model.initial_conditions[V_cell] = new_model.param.ocv_init
+        new_model.initial_conditions[V_cell] = 0
```

Comment:
just rename these to be more generic then we are good to go

Reply:
i.e. `target_variable` instead of `V_cell`, "Target variable [V]" instead of "Voltage variable [V]", etc